api/http/server.go

```diff
@@ -249,6 +254,11 @@ func (s *Server) HandleBzzGet(w http.ResponseWriter, r *http.Request) {
 				respondError(w, r, err.Error(), http.StatusUnauthorized)
 				return
 			}
+			if isRecoveryAttemptError(err) {
+				w.Header().Set("WWW-Authenticate", fmt.Sprintf("Basic realm=%q", uri.Address().String()))
+				respondError(w, r, err.Error(), http.StatusPaymentRequired)
```

Comment:
watch out, we need error `420`, not `402` (i believe this status isn't in the codebase, so you will need to add it).

we don't want users to pay _yet_ :wink:

Reply:
I looked at the code, and was really convinced I read 402 :smiley:
No worries I will not charge yet.
Changed it accordingly, thanks for spotting this. (facepalm moment)
`StatusRecoveryAttempt = 420`
...
`respondError(w, r, err.Error(), StatusRecoveryAttempt)`

Reply:
:+1: 

i would still name it `StatusEnhanceYourCalm` (going by the [unofficial but popular name](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#Unofficial_codes))

Reply:
changed to `StatusEnhanceYourCalm`